android/src/main/java/com/rudderstack/android/storage/AndroidStorageImpl.kt

```diff
@@ -115,7 +113,9 @@ class AndroidStorageImpl(
             DB_VERSION,
             providedExecutorService = storageExecutor
         )
-        messageDao = rudderDatabase?.getDao(MessageEntity::class.java, storageExecutor)
+        messageDao = rudderDatabase?.getDao(MessageEntity::class.java, storageExecutor).also {
+            Log.e("issue","messageDao created for v2: $it")
```

Comment:
We should be using logger object here:
```suggestion
            logger?.error(log = "messageDao created for v2: $it")
```

Reply:
It seems this was added for debugging purposes. If that's the case, please consider removing it.